std_msgs/Header header
float32 x_abs
float32 y_abs
float32 angle_abs
uint16 stuff_num